### DISCUSSION TITLE
Motion List, Volume and grid separation

### DISCUSSION THREAD

**Discussion by Quides5** [BaPSF/bapsf_motion#2] [Ideas]
I had a question regarding the gridding. Say we define a rectangle, and say "circular gridding" with some dr, dtheta

Do we make it so that it gives a circle, that fills out as much of the rectangle as possible, or a circle inscribed within rectangle

And do we want to make it so that the center of the grid is adjustable? Not sure how that would fit in, as the most natural grid origin should be the "centre" of whatever shape one draws.

**Comment by Quides5** [comment]
Basically, circumscribed grids or inscribed grids or customizable with default being former or latter.

**Comment by rocco8773** [comment]
I always views the shape/volume as a bounding volume on an infinite grid, or flip that around and the grid completely fills the defined shape/volume.  The approach always felt more logical to me, but I'm open to arguments otherwise.

Every grid pattern needs a starting point, and it makes sense that this "starting point" defaults to the center/origin of the bounding volume.  However, I do think the user should be able to adjust this starting point to what they want.
